<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBoxVisible">
        <property name="title">
         <string>Visible</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="checkBoxPieces">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Pieces</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxChessboard">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Chessboard</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>View</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButtonPlayer1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Player 1</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPlayer2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Player 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonCrowd">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Crowd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPawn">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Pawn</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxCamera">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Camera</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBoxEye">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Eye</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxEyeX">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
              <property name="value">
               <double>-4.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxEyeY">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxEyeZ">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
              <property name="value">
               <double>4.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxLookAt">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Look At</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxLookAtX">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxLookAtY">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxLookAtZ">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxViewUp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>View Up Vector</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxViewUpX">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
              <property name="value">
               <double>-4.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxViewUpY">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxViewUpZ">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="PbGLWidget" name="widgetGLWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="horizontalSliderRotationAngle">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="spinBoxRotationAngle">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelRotationAngle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Rotation Angle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PbGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">PbGLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonCrowd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>791</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonPawn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonPlayer2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonPlayer1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxEyeX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>753</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxEyeY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxEyeZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxLookAtZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxLookAtY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxLookAtX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxViewUpX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxViewUpY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxViewUpZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>1022</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxChessboard</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateVisible()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>842</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxPieces</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateVisible()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>881</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRotationAngle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>rotateCamera(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRotationAngle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxRotationAngle</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxRotationAngle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderRotationAngle</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxRotationAngle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>rotateCamera(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateCamera()</slot>
  <slot>updateView()</slot>
  <slot>updateVisible()</slot>
  <slot>rotateCamera()</slot>
  <slot>rotateCamera(int)</slot>
 </slots>
</ui>
